src/modules/paging/all.js

```diff
@@ -1,24 +1,41 @@
-import { flatten, range, isNil } from 'lodash';
+import { flatten, range, isNil, isEmpty } from 'lodash';
 import { getConfig } from '@globality/nodule-config';
 import { MaxLimitReached } from '../../error';
 import concurrentPaginate from '../concurrency';
 
 const DEFAULT_LIMIT = 20;
 
+function buildSearchRequestParams(searchArgs, limit, offset, sendBodyToSearchRequest) {
+    // Deliver params according to target search request - in args or in body
+    const paramsValues = {
+        ...searchArgs,
+        limit,
+        offset,
+    };
+    const paramsWrapper = (sendBodyToSearchRequest) ? { body: paramsValues } : paramsValues;
+    return paramsWrapper;
+}
+
 export default async function all(
```

Comment:
To be honest I would probably prefer to have two separate functions, e.g. "allForSavedSearch: and normal "all"  or something (which maybe called a common function) rather than adding logic in one function to handle these cases. I realize that what you are doing is probably more common in JS but I think it's more error prone and less clear.